src/lib/feature-finder.js

```diff
@@ -22,10 +22,27 @@ export default function(cucumberOptions) {
       let featureData = parseFeature(file);
```

Comment:
This is how a parsed feature with a scenario outline with 2 examples looks like:

```
{
  "type": "Feature",
  "tags": [],
  "location": {
    "line": 1,
    "column": 1
  },
  "language": "en",
  "keyword": "Feature",
  "name": "X",
  "children": [
    {
      "type": "ScenarioOutline",
      "tags": [
      ],
      "location": {
        "line": 7,
        "column": 3
      },
      "keyword": "Scenario Outline",
      "name": "I am an outline",
      "steps": [
        {
          "type": "Step",
          "location": {
            "line": 50,
            "column": 3
          },
          "keyword": "Then ",
          "text": "xxxx"
        }
      ],
      "examples": [
        {
          "type": "Examples",
          "tags": [],
          "location": {
            "line": 52,
            "column": 5
          },
          "tableBody": [
            {
              "type": "TableRow",
              "location": {
                "line": 54,
                "column": 7
              },
              "cells": [
                {
                  "type": "TableCell",
                  "location": {
                    "line": 54,
                    "column": 13
                  },
                  "value": "x"
                }
              ]
            },
            {
              "type": "TableRow",
              "location": {
                "line": 55,
                "column": 7
              },
              "cells": [
                {
                  "type": "TableCell",
                  "location": {
                    "line": 55,
                    "column": 13
                  },
                  "value": "x"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}
```